src/components/calcite-alert/calcite-alert.tsx

```diff
@@ -275,8 +294,8 @@ export class CalciteAlert {
   private determineActiveAlert(): void {
     if (this.queue?.[0] === this.el) {
       this.openAlert();
-      clearTimeout(this.autoDismissTimeout);
-      if (this.autoDismiss) {
+      if (this.autoDismiss && !this.autoDismissTimeout) {
+        this.trackTimer = new Date();
```

Comment:
To simplify, you can use `Date.now()` instead of creating a dates + calling `valueOf`.

Reply:
nice catch 🏈